homeassistant/components/bitvavo/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "domain": "bitvavo",
+    "name": "Bitvavo",
+    "config_flow": true,
+    "documentation": "https://www.home-assistant.io/integrations/bitvavo",
+    "requirements": ["bitvavo-aio==1.0.2"],
```

Comment:
The upstream package pins aiohttp strict at `==3.7.4`, this is something that cannot be used with Home Assistant. For example, we are at `==3.7.4.post0` and thus incompatible according to `pip`.

It is advised to use a version range instead. e.g.: `>=3.7.0`